USAGE.md

```diff
@@ -546,7 +546,7 @@ Each user can create up to 25 different suppression groups.
     Request request = new Request();
     request.setMethod(Method.POST);
     request.setEndpoint("asm/groups");
-    request.setBody("{\"is_default\":true,\"description\":\"Suggestions for products our users might like.\",\"name\":\"Product) Suggestions\"}";
+    request.setBody("{\"is_default\":true,\"description\":\"Suggestions for products our users might like.\",\"name\":\"Product) Suggestions\"}");
```

Comment:
This pattern is repeated throughout this file and unfortunately, it means that the examples here don't work / compile without this change.

I am not confirming that they work with this change, but, they would have a chance ...